evm-tests/src/subtensor.ts

```diff
@@ -338,6 +339,41 @@ export async function rootRegister(api: TypedApi<typeof devnet>, ss58Address: st
         hotkey: ss58Address
     })
 
+    await waitForTransactionCompletion(api, tx, singer)
+        .then(() => { })
+        .catch((error) => { console.log(`transaction error ${error}`) });
+}
+
+export async function setSubtokenEnable(api: TypedApi<typeof devnet>, netuid: number, subtokenEnable: boolean) {
+    const singer = getAliceSigner()
+    let internalTx = api.tx.AdminUtils.sudo_set_subtoken_enabled({
+       netuid: netuid,
+       subtoken_enabled: subtokenEnable
+    })
+    let tx = api.tx.Sudo.sudo({ call: internalTx.decodedCall })
+
+    await waitForTransactionCompletion(api, tx, singer)
+        .then(() => { })
+        .catch((error) => { console.log(`transaction error ${error}`) });
+
+}
+
+export async function startCall(api: TypedApi<typeof devnet>, netuid: number, keypair: KeyPair) {
+    const registerBlock = Number(await api.query.SubtensorModule.NetworkRegisteredAt.getValue(netuid))
+    let currentBlock = await api.query.System.Number.getValue()
+    const duration = Number(await api.constants.SubtensorModule.DurationOfStartCall)
+
+    while (currentBlock - registerBlock <= duration) {
```

Comment:
Isn't the registerBlock supposed to be after the currentBlock? Which means `currentBlock - registerBlock` would be negative and the condition will always be true?